sycl/test/regression/fsycl-save-temps.cpp

```diff
@@ -0,0 +1,23 @@
+//==--------------- fsycl-save-temps.cpp -----------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// Verify that a sample compilation succeeds with -save-temps
+// RUN: %clangxx -fsycl -save-temps %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
```

Comment:
Why do you need to actually run the program?

Reply:
Looks like an uber-safeguard, will remove.